src/System.Data.Common/src/System/Data/Common/DbConnectionOptions.cs

```diff
@@ -18,14 +18,13 @@ internal partial class DbConnectionOptions
 
         // differences between OleDb and Odbc
         // ODBC:
-        //     http://msdn.microsoft.com/library/default.asp?url=/library/en-us/odbc/htm/odbcsqldriverconnect.asp
-        //     http://msdn.microsoft.com/library/default.asp?url=/library/en-us/odbcsql/od_odbc_d_4x4k.asp
+        //     https://docs.microsoft.com/en-us/sql/odbc/reference/syntax/sqldriverconnect-function
         //     do not support == -> = in keywords
         //     first key-value pair wins
         //     quote values using \{ and \}, only driver= and pwd= appear to generically allow quoting
         //     do not strip quotes from value, or add quotes except for driver keyword
         // OLEDB:
-        //     http://msdn.microsoft.com/library/default.asp?url=/library/en-us/oledb/htm/oledbconnectionstringsyntax.asp
+        //     https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/connection-string-syntax#oledb-connection-string-syntax
```

Comment:
You're correct, it doesn't really help a lot. Maybe https://docs.microsoft.com/en-us/dotnet/api/system.data.oledb.oledbconnection.connectionstring is more appropriate, since it talks about the actual structure of how a connectionstring should look like?